seed/challenges/bootstrap.json

```diff
@@ -84,8 +84,16 @@
       "descriptionFr": [],
       "nameRu": "",
       "descriptionRu": [],
-      "nameEs": "",
-      "descriptionEs": [],
+      "nameEs": "Usa diseño adaptativo con los contenedores fluidos de Bootstrap",
+      "descriptionEs": [
+        "Ahora vamos de vuelta a nuestro App de fotos de gatos. Esta vez, vamos a darle estilo utilizando la infraestructura CSS del popular Bootstrap.",
+        "Bootstrap determina qué tan ancha es la pantalla y adapta correspondientemente el tamaño de tus elementos HTML - por eso es que se llama <code>Diseño adaptativo</code>.",
```

Comment:
¿Para relacionar con el término en inglés, que tal agregar al final entre paréntesis  y en itálicas (<em>responsive design</em>) ?